kbfstool/md_dump.go

```diff
@@ -154,7 +142,12 @@ Branch can be:
     the ID of the master branch "00000000000000000000000000000000", or
   - omitted, in which case it is treated as if it were the string "device";
 
-and Revision can be:
+and RevisionRange can be in the following format:
+
+  Revision
+  Revision-Revision
```

Comment:
Reading through the code, it seems like it's intended to support queries where rev2 < rev1, and the MDs are dumped in reverse order in that case.  Would be good to document that here, so people aren't surprised by it.

Reply:
Done.